nengo_gui/examples/tutorial/25-spa-parse.py

```diff
@@ -0,0 +1,26 @@
+# Tutorial 25: Parsing Simple Commands
```

Comment:
Should behave better now...  I've gotten rid of the constants and added a description.  :)